com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -1,15 +1,15 @@
 id: com.valvesoftware.Steam.CompatibilityTool.Proton-GE
 default-branch: stable
 runtime: com.valvesoftware.Steam
-runtime-version: stable
-sdk: org.freedesktop.Sdk//21.08
+runtime-version: beta
+sdk: org.freedesktop.Sdk//22.08beta
 build-extension: true
 appstream-compose: false
 
 sdk-extensions:
   - org.freedesktop.Sdk.Compat.i386
   - org.freedesktop.Sdk.Extension.toolchain-i386
-  - org.freedesktop.Sdk.Extension.mingw-w64
+  - org.freedesktop.Sdk.Extension.mingw-w64//beta
```

Comment:
You don't need to specify sdk extension branch here. Besides, `beta` is wrong, it's `22.08beta` (i.e. matches the sdk).

Reply:
Anyway, specifying the branch seems to not be supported by flatpak-builder.